docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,278 @@ task.download_outputs()
+
+### Post-Simulation
+
+After the simulation completes, the output files will be downloaded locally by
+calling `task.download_outputs()`. The outputs will include simulation results
+like mesh quality reports, final results, and log files for each step in the process.
+
+To inspect the output, look under the `inductiva_output` folder where the
+results are downloaded:
+
+```bash
+ls -lasgo inductiva_output/task_id
+total 672
+  0 drwxr-xr-x  143     4576 Sep 19 16:06 .
+  0 drwxr-xr-x  400    12800 Sep 19 16:05 ..
+  0 drwxr-xr-x   11      352 Sep 19 16:06 0.orig
+  8 -rw-r--r--    1      626 Sep 19 16:06 Allclean
+ 16 -rw-r--r--    1     6998 Sep 19 16:06 Allrun
+  8 -rw-r--r--    1      991 Sep 19 16:06 COPYING
+ 48 -rw-r--r--    1    21547 Sep 19 16:06 README.md
+  0 -rw-r--r--    1        0 Sep 19 16:06 case.foam
+  0 drwxr-xr-x    8      256 Sep 19 16:06 constant
+  0 drwxr-xr-x   27      864 Sep 19 16:06 dynamicCode
+  0 drwxr-xr-x   13      416 Sep 19 16:06 figures
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor0
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor1
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor10
+    ...
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor126
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor127
+ 32 -rw-r--r--    1    14437 Sep 19 16:06 stderr.txt
+536 -rw-r--r--    1   272331 Sep 19 16:06 stdout.txt
+  0 drwxr-xr-x   30      960 Sep 19 16:06 system
+ 24 -rw-r--r--    1    11399 Sep 19 16:06 thumbnail.png
+```
+
+You can now perform post-processing on the results, just as you would if you had
+run the simulation locally.
+
+### Small benchmark
+
+We conducted a series of OpenFOAM simulations across different machine
+configurations to evaluate the performance impact of varying core counts and the
+effect of hyperthreading. The configurations ranged from 90 to 360 virtual CPUs
+(vCPUs), with some tests utilizing hyperthreading while others disabled it. Our
+goal was to assess how these hardware differences influence simulation time.
+
+| Machine Configuration            | nCores/n_vCPUs           | Hyperthreading Status        | Time (min:sec) |
+|-----------------------------------|--------------------------|------------------------------|----------------|
+| c3d-standard-90                   | 90                       | Enabled                      | 7:48           |
+| c3d-standard-180                  | 180                      | Enabled                      | 7:39           |
+| c3d-standard-360                  | 360                      | Enabled                      | 8:32           |
+| c3d-standard-90                   | 45                       | Disabled (Hyperthreading Off) | 7:20           |
+| c3d-standard-180                  | 90                       | Disabled (Hyperthreading Off) | 6:46           |
+| c3d-standard-360                  | 180                      | Disabled (Hyperthreading Off) | 6:57           |
```

Comment:
are these results with hyperthreding disabled (`threads_per_core` = 1) or just using using half of the total number of available vcpus?